modules/apps/27-interchain-accounts/keeper/relay_test.go

```diff
@@ -0,0 +1,83 @@
+package keeper_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	ibctesting "github.com/cosmos/ibc-go/testing"
+)
+
+func (suite *KeeperTestSuite) TestTrySendTx() {
```

Comment:
I know GetActiveChannel should not return a channel id that doesn't exist, but do we want to add a test case for this to ensure our `GetChannel` function would catch this if we have a bug in our active channel mapping?

This could be done by setting the active channel for a port id to a channel that doesn't exist (in the malleate function)

Should we add a test case for the error return by SerializeCosmosTx? Maybe define a new message type not registered on the cdc?

Reply:
Good idea. Updated :+1: